PersonsApi/Swagger/SwaggerOptions.cs

```diff
@@ -0,0 +1,63 @@
+
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace PersonsApi.Swagger
+{
+    public class SwaggerOptions : IConfigureNamedOptions<SwaggerGenOptions>
+    {
+        private readonly IApiVersionDescriptionProvider _provider;
+
+        private const string ApiKeyName = "ApiKey";
+        private const string ServiceTitle = "Persons API";
+        private const string ServiceDescription = "The Academies API provides users with access to a variety of data " +
```

Comment:
This description is for the Academies API and should be updated for the Persons API